server/pom.xml

```diff
@@ -47,12 +47,11 @@
         <maven.coveralls.plugin.version>4.3.0</maven.coveralls.plugin.version>
         <maven.jacoco.plugin.version>0.8.5</maven.jacoco.plugin.version>
         <maven.resources.plugin.version>3.0.2</maven.resources.plugin.version>
-        <maven.source.plugin.version>3.0.1</maven.source.plugin.version>
         <maven.surefire.plugin.version>3.0.0-M3</maven.surefire.plugin.version>
         <mockito.core.version>2.23.0</mockito.core.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <revision>0.13.0+NEXT</revision>
+        <revision>1.0.0-SNAPSHOT</revision>
```

Comment:
Does this mean the server code won't be versioned in the repository anymore? Version would be only put there at build time?

Reply:
We decided not to version our java part. We don't expose our jar in public and use it only internally. So the end-user in not care what exact java part version he have. And also no one has a way to download some particular java part version separately.
That's why we decided to shrink our effort to keep all versions updated.

Reply:
👍 🆗